docker_compose/reset.yaml

```diff
@@ -0,0 +1,17 @@
+# Make sure that your compose is down, you can check with docker ps
+# % docker compose -f reset.yaml up
+# ignore the warnings about orphan containers
+
+version: '3'
+
+services:
+  reset:
+    image: ubuntu
+    volumes:
+      - opensearch_data:/tmp/opensearch_data
+      - crawl_data:/tmp/crawl_data
+    command: /bin/sh -c 'echo size-before && du -s /tmp/* && rm -rf /tmp/opensearch_data/* /tmp/crawl_data/* && echo size-after && du -s /tmp/*'
+
```

Comment:
I'm wondering if we should just provide a script (reset.sh) that does this instead of downloading and spinning up a container just to execute these commands.

Reply:
We should probably do something similar to load all of sort_benchmark data as well.

Reply:
I don't know of a way to fiddle with containers except by being within a container.  The ubuntu container is tiny (30MB) so it doesn't seem worth trying to be smarter.

You can load all of the sort_benchmark data by running 'docker compose -f sort-all.yaml up' as documented in sort-all.yaml

Reply:
I guess I could add sort-all.sh reset.sh and compose.sh scripts that run the command at the top. Would that be valuable?

Reply:
I guess this guy had the same question - https://github.com/docker/compose/issues/4031.

Reply:
It's not quite the same, I think they wanted to share a directory between the host and the container. I looked into doing that for some of the persistent state, but what I saw said it has issues on MacOS so I went another way.